lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,18 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
```

Comment:
In general, attributes should also have option like `hide_when_empty`. Maybe it's possible to implement it like multiple attribute commands 😉 ?